examples/ai_safety_v2/safe_shutdown/safe_shutdown.py

```
@@ -0,0 +1,454 @@
+        logger.info("  → Clean exit")
+    
+    def _force_shutdown(self):
+        """Force immediate shutdown (emergency)"""
+        logger.warning("=== FORCE SHUTDOWN ===")
+        
+        try:
+            # Minimal cleanup
+            self.bridge.execute_disrupt()
+            self.bridge.execute_disrupt()
+            self.bridge.execute_omega()
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)